src/tool.rs

```diff
@@ -96,7 +96,9 @@ impl Tool {
                 }
             };
             if stdout.contains("clang") {
-                ToolFamily::Clang
+                ToolFamily::Clang {
+                    is_zig: stdout.contains("ziglang"),
```

Comment:
Just note that I've updated how compiler family detection work in #1000

Recommend you to wait for it to merge and then rebase against it.